client/src/App.tsx

```diff
@@ -19,21 +20,8 @@ function App() {
             </main>
           }
         />
-        <Route
-          path="*"
-          element={
-            <main>
-              <Header />
-              <div className="flex min-h-[calc(100vh-4.5rem)] items-center justify-center">
-                <h1 className="rounded-xl border bg-gray-200 px-8 py-4 text-lg font-bold text-gray-700 shadow">
-                  Page Not Found
-                </h1>
-              </div>
-            </main>
-          }
-        />
-        <Route path="/role-assignment"
-          element={<RoleAssignmentPage />} />
+        <Route path="/role-assignment" element={<RoleAssignmentPage />} />
```

Comment:
Minor point but do we want to call NoFound or NoFoundPage? I now feel like NoFound sounds better. If so, from now on whenever we create a new page, we can just name its file and component without appending page in the end.

Reply:
Yes I feel like adding page at the end is redundant since the parent file is called pages. What do you guys think? @Josh-D18 @jaydelim

Reply:
I did my "NotFound" [SPE-18] and "Landing" [SPE-14] pages with this naming scheme